visualization/app/codeCharta/util/customConfigHelper.ts

```diff
@@ -139,7 +138,8 @@ export class CustomConfigHelper {
 				customConfigVersion: exportedConfig.customConfigVersion,
 				mapSelectionMode: exportedConfig.mapSelectionMode,
 				stateSettings: exportedConfig.stateSettings,
-				camera: exportedConfig.camera
+				camera: exportedConfig.camera,
+				note: exportedConfig.note
```

Comment:
I think this could break if a config without the new "note" property is uploaded.
We should safely handle this case, if not already done.

Reply:
I have adjusted the 'note' property. Now it is optional and is only set when a note has been set by the user. We no longer have empty strings and backward compatibility problems.